File: TweetJournalApi/Contracts/V1/Responses/EntryResponse.cs
using System;

namespace TweetJournalApi.Contracts.V1.Responses
{
    public class EntryResponse
    {
        public Guid Id { get; set; }
        public string Content { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime ModifiedDate { get; set; }

    }
}

File: TweetJournal.Web/wwwroot/old/Contract/Responses/EntryResponse.js
// eslint-disable-next-line require-jsdoc
export class EntryResponse {
  constructor ({ id = '', content = '', createdDate = '', modifiedDate = '' }) {
    this.id = id
    this.content = content
    this.createdDate = createdDate
    this.modifiedDate = modifiedDate
  }
}
